app/kubernetes/react/views/index.ts

```diff
@@ -20,15 +19,4 @@ export const viewsModule = angular
   .component(
     'kubernetesIngressesCreateView',
     r2a(withUIRouter(withReactQuery(withCurrentUser(CreateIngressView))), [])
-  )
-  .component(
-    'beOnlyButton',
-    r2a(withUIRouter(withReactQuery(withCurrentUser(BEOnlyButton))), [
```

Comment:
Should the new r2a beOnlyButton in app/portainer/react/components/index.ts need these withUIRouter, withReactQuery, withCurrentUser wrapper components? Removing them could risk introducing new bugs.

Reply:
As discussed, I think it's fine.  Have renamed beOnlyButton to BETeaserButton to be more inline with what we call "Teaser" in other places